source/_components/watson_iot.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: "IBM Watson IoT Platform"
+description: "Record events in the IBM Watson IoT Platform."
+date: 2018-04-03 22:09
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: History
+ha_release: 0.67
+---
+
+The `watson_iot` component enables you to link the devices in home-assistant
+with an [IBM Watson IoT Platform instance](https://www.ibm.com/us-en/marketplace/internet-of-things-cloud).
+
+## {% linkable_title Configuration %}
+
+To use this component, you first need to register a gateway device type and then
+a gateway device in your IoT platform instance. For instructions on how to do
+this check the [official documentation](https://console.bluemix.net/docs/services/IoT/gateways/dashboard.html#IoT_connectGateway)
+which provides the details on doing this. After you register the gateway device
+for your home-assistant you'll need 5 pieces of information:
+ - Organization ID
+ - Gateway device Type
+ - Gateway device ID
+ - Authentication Token
+
+With this basic information you can configure the component:
+
+```yaml
+# Example configuration.yaml entry:
+watson_iot:
+  organization: 'organization_id'
+  type: 'device_type'
+  id: 'device_id'
+  token: 'auth_token'
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
Ok, I pushed a patch to use the configuration tags